src/course-libraries/CourseLibraries.tsx

```diff
@@ -0,0 +1,324 @@
+            spacer={<span className="custom-spacer">/</span>}
+            linkAs="span"
+          />
+        </Stack>
+      </Card.Section>
+    </Card>
+  );
+};
+
+const LibraryCard: React.FC<LibraryCardProps> = ({ courseId, title, links }) => {
+  const intl = useIntl();
+  const linksInfo = useMemo(() => keyBy(links, 'downstreamUsageKey'), [links]);
+  const totalComponents = links.length;
+  const outOfSyncCount = useMemo(() => countBy(links, 'readyToSync').true, [links]);
+  const downstreamKeys = useMemo(() => uniq(Object.keys(linksInfo)), [links]);
+  const { data: downstreamInfo } = useFetchIndexDocuments(
+    [`context_key = "${courseId}"`, `usage_key IN ["${downstreamKeys.join('","')}"]`],
+    downstreamKeys.length,
+    ['usage_key', 'display_name', 'breadcrumbs', 'description', 'block_type'],
+    ['description:30'],
+    [SearchSortOption.TITLE_AZ],
+  ) as unknown as { data: ComponentInfo[] };
+
+  const renderBlockCards = (info: ComponentInfo) => {
+    // eslint-disable-next-line no-param-reassign
+    info.readyToSync = linksInfo[info.usageKey].readyToSync;
+    return <BlockCard info={info} key={info.usageKey} />;
+  };
+
+  return (
+    <Collapsible.Advanced>
+      <Collapsible.Trigger className="bg-white shadow px-2 py-2 my-3 collapsible-trigger d-flex font-weight-normal text-dark">
+        <Collapsible.Visible whenClosed>
+          <Icon src={KeyboardArrowRight} />
+        </Collapsible.Visible>
+        <Collapsible.Visible whenOpen>
+          <Icon src={KeyboardArrowDown} />
+        </Collapsible.Visible>
+        <Stack direction="vertical" className="flex-grow-1 pl-2 x-small" gap={1}>
+          <h4>{title}</h4>
+          <Stack direction="horizontal" gap={2}>
+            <span>
+              {intl.formatMessage(messages.totalComponentLabel, { totalComponents })}
+            </span>
+            <span>/</span>
+            {outOfSyncCount ? (
+              <>
+                <Icon src={Loop} size="xs" />
+                <span>
+                  {intl.formatMessage(messages.outOfSyncCountLabel, { outOfSyncCount })}
+                </span>
+              </>
+            ) : (
+              <>
+                <Icon src={CheckCircle} size="xs" />
+                <span>
+                  {intl.formatMessage(messages.allUptodateLabel)}
+                </span>
+              </>
+            )}
+          </Stack>
+        </Stack>
+        <Dropdown onClick={(e: { stopPropagation: () => void; }) => e.stopPropagation()}>
+          <Dropdown.Toggle
+            id={`dropdown-toggle-${title}`}
+            alt="dropdown-toggle-menu-items"
+            as={IconButton}
+            src={MoreVert}
+            iconAs={Icon}
+            variant="primary"
+          />
+          <Dropdown.Menu>
+            <Dropdown.Item>TODO 1</Dropdown.Item>
+            <Dropdown.Item>TODO 2</Dropdown.Item>
+          </Dropdown.Menu>
```

Comment:
@navinkarkera I think you need to disable this menu until functionality is added